aten/src/ATen/native/cuda/TensorCompare.cu

```diff
@@ -30,9 +34,47 @@ void where_kernel_impl(TensorIterator &iter, ScalarType condition_type) {
   });
 }
 
+void isposinf_kernel_impl(TensorIterator &iter) {
+  if (c10::isIntegralType(iter.input_dtype(), /*include_bool=*/true)) {
+    AT_DISPATCH_INTEGRAL_TYPES_AND(at::ScalarType::Bool, iter.input_dtype(), "isposinf_cuda", [&]() {
+      gpu_kernel(
+        iter,
+        [] GPU_LAMBDA (scalar_t a) -> bool { return false; }
+      );
+    });
+  } else {
+    AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.input_dtype(), "isposinf_cuda", [&]() {
```

Comment:
Same question about bfloat16 dispatch as above.

Reply:
In my local dev environment, the bloat16 dispatch calls `c10::BFloat16 infinity()` of `pytorch/c10/util/BFloat16-inl.h`.